NchanSubscriber.js

```diff
@@ -798,8 +806,14 @@
           if(incoming) { this.opt.headers[name]= incoming; }
         }, this);
 
-        this.reqStartTime = new Date().getTime();
-        
+        this.pollingRequest = ughbind(function() {
```

Comment:
I've created this pollingRequest callback since the amount of logic to-do before creating the longpoll event was starting to pile up.